ristretto/snapcraft.yaml

```diff
@@ -19,10 +19,15 @@ parts:
     plugin: autotools
     configflags:
       - --enable-maintainer-mode  # so src/marshal.h gets generated
-  env:
-    plugin: copy
-    files:
-      wrapper: usr/bin/ristretto.wrapper
+    build-packages:
```

Comment:
Maybe let's move it. It might make sense in a situation where you want to distinguish between build-packages which have to be pulled in for different parts.